handler/src/main/java/io/netty/handler/ipfilter/UniqueIpFilter.java

```diff
@@ -38,10 +38,9 @@
     @Override
     protected boolean accept(ChannelHandlerContext ctx, InetSocketAddress remoteAddress) throws Exception {
         final InetAddress remoteIp = remoteAddress.getAddress();
-        if (connected.contains(remoteIp)) {
+        if (!connected.add(remoteIp)) {
```

Comment:
@normanmaurer not a big deal but don't you usually prefer guard statements to avoid code branches?

Reply:
Depends... I think it’s fine here